cms/pytest.ini

```diff
@@ -1,4 +1,8 @@
 [pytest]
+# Note: pytest settings are read here, but also in setup.cfg.
```

Comment:
I feel like something like this might be a little less misleading:
```suggestion
# Note: these pytest settings completely override those in setup.cfg.
```

Reply:
Good points.  I've made the comment even more specific.

Reply:
I still don't like the initial sentence, because it is misleading if someone doesn't read on. But I like your new comment. What if you drop the first sentence, and update the second sentence to read:
```
# Note: The first file of settings found is used, there is no combining, so this file is
...
```